### ISSUE TITLE
Question about  chestmnist dataset

### ISSUE THREAD

**Issue by nightrain-vampire** [MedMNIST/MedMNIST#46]
When I use the chestmnist dataset, I found:
```
class c = 0: 70472 real images
class c = 1: 7996 real images
class c = 2: 0 real images
class c = 3: 0 real images
class c = 4: 0 real images
class c = 5: 0 real images
class c = 6: 0 real images
class c = 7: 0 real images
class c = 8: 0 real images
class c = 9: 0 real images
class c = 10: 0 real images
class c = 11: 0 real images
class c = 12: 0 real images
class c = 13: 0 real images
```
However, it seems that the chestmnist dataset has multi-label:
```
Dataset ChestMNIST of size 28 (chestmnist)
    Number of datapoints: 78468
    Root location: /home/user3/.medmnist
    Split: train
    Task: multi-label, binary-class
    Number of channels: 1
    Meaning of labels: {'0': 'atelectasis', '1': 'cardiomegaly', '2': 'effusion', '3': 'infiltration', '4': 'mass', '5': 'nodule', '6': 'pneumonia', '7': 'pneumothorax', '8': 'consolidation', '9': 'edema', '10': 'emphysema', '11': 'fibrosis', '12': 'pleural', '13': 'hernia'}
    Number of samples: {'train': 78468, 'val': 11219, 'test': 22433}
    Description: The ChestMNIST is based on the NIH-ChestXray14 dataset, a dataset comprising 112,120 frontal-view X-Ray images of 30,805 unique patients with the text-mined 14 disease labels, which could be formulized as a multi-label binary-class classification task. We use the official data split, and resize the source images of 1×1024×1024 into 1×28×28.
    License: CC BY 4.0
===================
Dataset ChestMNIST of size 28 (chestmnist)
    Number of datapoints: 22433
    Root location: /home/user3/.medmnist
    Split: test
    Task: multi-label, binary-class
    Number of channels: 1
    Meaning of labels: {'0': 'atelectasis', '1': 'cardiomegaly', '2': 'effusion', '3': 'infiltration', '4': 'mass', '5': 'nodule', '6': 'pneumonia', '7': 'pneumothorax', '8': 'consolidation', '9': 'edema', '10': 'emphysema', '11': 'fibrosis', '12': 'pleural', '13': 'hernia'}
    Number of samples: {'train': 78468, 'val': 11219, 'test': 22433}
    Description: The ChestMNIST is based on the NIH-ChestXray14 dataset, a dataset comprising 112,120 frontal-view X-Ray images of 30,805 unique patients with the text-mined 14 disease labels, which could be formulized as a multi-label binary-class classification task. We use the official data split, and resize the source images of 1×1024×1024 into 1×28×28.
    License: CC BY 4.0
```
How can I use the multi-label instead of just binary-class?

**Comment by ThreeSRR** [comment]
Hi @nightrain-vampire， 

For multi-label binary-class datasets, the labels are represented in an array with shape `N x L`, where `L` denotes the number of task labels in the multi-label dataset. Hence, the label array of chestmnist is with shape `N x 14`. 

In other words, each sample has a label of 14 binary values, denoting positive (1) or negative (0) for each of the 14 diseases. You can use the following code to check the labels:

```python
train_dataset = medmnist.ChestMNIST(split='train', download=True)
print(train_dataset.labels.sum(axis=0))
```

**Comment by nightrain-vampire** [comment]
> Hi @nightrain-vampire，
> 
> For multi-label binary-class datasets, the labels are represented in an array with shape `N x L`, where `L` denotes the number of task labels in the multi-label dataset. Hence, the label array of chestmnist is with shape `N x 14`.
> 
> In other words, each sample has a label of 14 binary values, denoting positive (1) or negative (0) for each of the 14 diseases. You can use the following code to check the labels:
> 
> ```python
> train_dataset = medmnist.ChestMNIST(split='train', download=True)
> print(train_dataset.labels.sum(axis=0))
> ```

I understand! Thank you very much!
